server/sonar-db-dao/src/test/java/org/sonar/db/qualityprofile/QualityProfileDbTester.java

```diff
@@ -33,6 +36,20 @@ public QualityProfileDbTester(DbTester db) {
     this.dbSession = db.getSession();
   }
 
+  /**
+   * Create a profile with random field values, except default=false, on the specified organization.
+   */
+  public QualityProfileDto insert(OrganizationDto organization, Consumer<QualityProfileDto>... consumers) {
+    QualityProfileDto profile = QualityProfileTesting.newQualityProfileDto()
+      .setDefault(false)
```

Comment:
in fact I'm not sure that it's good idea to keep this hardcoded behavior. I will drop it.